db.php

```diff
@@ -0,0 +1,8 @@
+<?php
+$db = new PDO('mysql:host=db; dbname=collector', 'root', 'password');
+$db->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+$query = $db->prepare('SELECT * FROM `exoplanets`');
+$query->execute();
+$exoPlanets = $query->fetchAll();
```

Comment:
Better to move this query to `index.php`, otherwise it will also run when you `require_once 'db.php'` in your `add_item.php` as well, where it isn't needed.